x-govuk/components/secondary-navigation/template.njk

```diff
@@ -0,0 +1,14 @@
+<nav class="x-govuk-secondary-navigation {%- if params.classes %} {{ params.classes }}{% endif -%}" aria-label="Sub navigation" {%- for attribute, value in params.attributes %} {{ attribute }}="{{ value }}"{% endfor %}>
```

Comment:
We should follow the example of the primary navigation component and label the navigation with a header. Or maybe we stick with `aria-label` in this case, but the default value should be something akin to `Sub sections`, but either way it shouldn’t include the word ‘navigation’ as that’s implicit and will be announced already because of the navigation role.

However, the example you include in the guidance is a good example where you’d want to have the navigation labelled by another heading or element on the page already (i.e. ‘James Smith’). Maybe we provide an option to override the value for `aria-labelledby`, and if that’s given we don’t add the visually hidden heading?

So, with all that in mind, something like this: 

```suggestion
<nav class="x-govuk-secondary-navigation {%- if params.classes %} {{ params.classes }}{% endif -%}" aria-labelledby="{{ params.labelledby or "secondary-navigation-heading" }}"{%- for attribute, value in params.attributes %} {{ attribute }}="{{ value }}"{% endfor %}>
{% if not params.labelledby %}
  <h2 class="govuk-visually-hidden" id="secondary-navigation-heading">{{ params.visuallyHiddenTitle or "Sub sections" }}</h2>
{% endif %}
```

Reply:
I've updated this in 7cbcd1f1ca333bd8f3fab4b00da3fb68ee231747 so that the `aria-label` text is set by `params.visuallyHiddenTitle` for consistency with the primary navigation.

Being able to set `aria-labelledby` value for an ID which is outside of the component itself is interesting - but feels error-prone?   I'm not enough of an accessibility expert to know whether it's better to use an `aria-label` or a reference to an existing heading.  Probably the whole thing needs a bit of an accessibility review.